SDK Bump to 1.0.73-alpha

Velin92 opened the pull request.

Velin92 commented on ElementX/Sources/Services/Client/ClientProxy.swift:
```diff
@@ -437,7 +437,7 @@ class ClientProxy: ClientProxyProtocol {
                 .addList(listBuilder: visibleRoomsListBuilder)
                 .withCommonExtensions()
-                .bumpEventTypes(bumpEventTypes: roomListRecencyOrderingAllowedEventTypes)
+//                .bumpEventTypes(bumpEventTypes: roomListRecencyOrderingAllowedEventTypes)
```
not sure about this @stefanceriu  this function seems to have been removed

stefanceriu replied:
Yes, as https://github.com/matrix-org/sliding-sync/pull/142 the bunp event types are per list now

Velin92 replied:
should it be only for this list right?

stefanceriu replied:
No no, as per the other comment here it needs to be on both visible and all

stefanceriu commented on ElementX/Sources/Services/Client/ClientProxy.swift:
```diff
@@ -466,12 +462,15 @@ class ClientProxy: ClientProxyProtocol {
             .requiredState(requiredState: slidingSyncRequiredState)
             .filters(filters: slidingSyncFilters)
             .syncModeSelective(selectiveModeBuilder: .init().addRange(start: 0, endInclusive: 20))
+            .bumpEventTypes(bumpEventTypes: roomListRecencyOrderingAllowedEventTypes)
```
Oh.. I think we need this in the allRoomsList as well otherwise their ordering will be different and merging the 2 lists will be a mess.

Velin92 replied:
done!

stefanceriu reviewed (changes requested):
Looks fine except the bump event types that need to be shared

Velin92 commented:
@stefanceriu  I have also removed all the code related to the local notifications since it's kind of a thing we will probably never need again or way later in the future, so to avoid confusion I removed the FF and the code related to it.
